src/components/tabs/PlotTab.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { ReactElement } from "react";
+
+import { Dataset, Track } from "../../colorizer";
+import { Input } from "antd";
+import { SearchOutlined } from "@ant-design/icons";
+import IconButton from "../IconButton";
+import PlotWrapper from "../PlotWrapper";
+import styled from "styled-components";
+import { FlexRowAlignCenter, NoSpinnerContainer } from "../../styles/utils";
+
+const TrackTitleBar = styled(FlexRowAlignCenter)`
+  justify-content: end;
+  flex-wrap: wrap;
+`;
+
+const TrackSearch = styled(FlexRowAlignCenter)`
+  & input {
+    max-width: 80px;
+  }
+
+  & h3 {
+    margin: 0;
+  }
+`;
+
+type PlotTabProps = {
+  findTrackInputText: string;
+  setFindTrackInputText: (text: string) => void;
+  disabled?: boolean;
+  findTrack: (trackId: number, seekToFrame?: boolean) => void;
+  currentFrame: number;
+  dataset: Dataset | null;
+  featureName: string;
+  selectedTrack: Track | null;
+};
+
+const defaultProps: Partial<PlotTabProps> = {
+  disabled: false,
```

Comment:
I feel like either all the props should have defaults set here or none of them should, and somewhere above all the defaults are set and sent in.